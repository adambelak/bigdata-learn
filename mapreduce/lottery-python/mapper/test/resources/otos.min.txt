2016;5;2016.02.06.;0;0 Ft;30;1 910 065 Ft;2955;20 530 Ft;86050;1 370 Ft;11;15;26;39;78
2016;4;2016.01.30.;0;0 Ft;16;3 384 925 Ft;1729;33 165 Ft;60555;1 840 Ft;48;55;62;69;80
2016;3;2016.01.23.;1;1 496 723 740 Ft;46;1 481 290 Ft;3797;19 000 Ft;107680;1 305 Ft;11;27;55;68;80