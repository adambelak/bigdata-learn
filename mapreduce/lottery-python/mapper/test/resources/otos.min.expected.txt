11	1
15	1
26	1
39	1
78	1
48	1
55	1
62	1
69	1
80	1
11	1
27	1
55	1
68	1
80	1